src/rest.ts

```diff
@@ -120,8 +139,8 @@ function connect(uid) {
                 }
             };
 
-            const onDataMessage = (event, message) => {
-                if (return_raw) {
+            const onDataMessage = (event: unknown, message: Uint8Array) => {
```

Comment:
Hmm, `event` is unused. Isn't that a linting error?

Reply:
Apparently not, maybe because it's a signature? Anyway, I marked it with `_`.